theme.css

```diff
@@ -2172,4 +2172,36 @@ li[data-task="b"]>input:checked,
 li[data-task="b"]>p>input:checked {
 	color: rgba(var(--color-orange-rgb), 1);
 	-webkit-mask-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' class='h-5 w-5' viewBox='0 0 20 20' fill='currentColor'%3E%3Cpath d='M5 4a2 2 0 012-2h6a2 2 0 012 2v14l-5-2.5L5 18V4z' /%3E%3C/svg%3E");
-}
\ No newline at end of file
+}
+
+
+/* Add Obsidian Icon on new Tab */
+/* Copied from Cupertino Theme :P */
+.empty-state-container {
```

Comment:
It created a bit of a mess

![image](https://github.com/user-attachments/assets/45692244-b499-4444-be1d-0755736868dc)